docs/content/en/docs/yaml-crd-ref/config.md

```diff
@@ -3,3 +3,56 @@ title: KeptnConfig
 description: Define configuration values
 weight: 20
 ---
+
+`KeptnConfig` defines configuration values for the Keptn Lifecycle Toolkit.
+Currently, it is used to configure the URL of the OpenTelemetry collector.
+
+## Yaml Synopsis
+
+```yaml
+apiVersion: options.keptn.sh/v?alpha?
+kind: KeptnConfig
+metadata:
+  name: <collector-name>
+spec:
+  OTelCollectorUrl: '<otelurl:port>'
+```
+
+## Fields
+
+* **apiVersion** -- API version being used.
+`
+* **kind** -- Resource type.
+   Must be set to `KeptnConfig`.`
+
+* **metadata**
+  * **name** -- Unique name of this collector.
+    This is the name that identifies this collector
+    in the `KeptnConfigList` CR.
+    * Must be an alphanumeric string and, by convention, is all lowercase.
+    * Can include the special characters `_`, `-`, (others?)
+    * Should not include spaces.
+
+## Usage
+
+The `KeptnConfigList` CR is generated automatically
+and contains a list of all valid `KeptnConfig` CR's.
```

Comment:
I don't think this info is needed at all. The list objects for all CRDs are basically just internal API objects that will never be seen directly by a user

Reply:
Removed

Reply:
deleted